### PR TITLE
feat: add  raw TrocadilhoList feature

### PR DESCRIPTION
**Pull request by bioneoficial**
image
[

- Add Trocadilho Component
- Add TrocadilhoList Component
- Add LikeButton Component
- Add Router

### CODE REVIEW

File: src/atoms/LikeButton/LikeButton.css [hunk 1]
```diff
@@ -0,0 +1,10 @@
+.likeButton{
+    color: white;
+    background-color: green;
+    border-radius: 5px;
+    padding: 0.6rem;
+    margin: 0.5rem;
+    cursor: pointer;
+    border: none;
+    width: max-content;
+}
```
Comment (julialbq) [inline src/atoms/LikeButton/LikeButton.css]: add line at the end of file

File: src/atoms/LikeButton/index.js [hunk 2]
```diff
@@ -0,0 +1,3 @@
+import LikeButton from "./LikeButton";
+
+export default LikeButton;
```
Comment (julialbq) [inline src/atoms/LikeButton/index.js:3]: add line at the end of file

File: src/atoms/LikeButton/LikeButton.js [hunk 3]
```diff
@@ -0,0 +1,10 @@
+import React from "react";
+import "./LikeButton.css";
+
+const LikeButton = () => {
+  return (
+      <button className="likeButton">DEIXE SEU LIKE!</button>
+  );
+}
+
+export default LikeButton;
```
Comment (julialbq) [inline src/atoms/LikeButton/LikeButton.js]: add line at the end of file

File: src/index.css [hunk 4]
```diff
@@ -1,13 +1,13 @@
...
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
 }
 
 code {
-  font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New',
+  font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
-}
+}
```
Comment (julialbq) [inline src/index.css]: add line at the end of file

File: src/organisms/PunList/PunList.js [hunk 5]
```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import Pun from "../../atoms/Pun";
+import "./PunList.css";
+import { PUNS } from "../../utils/enums";
+
+const PunList = () => {
+  localStorage.setItem("puns", JSON.stringify(PUNS));
+  const punsFromStorage = JSON.parse(localStorage.getItem("puns"));
```
Comment (julialbq) [inline src/organisms/PunList/PunList.js]: probably you're still going to do it (I didn't do it yet either), but the useLocalStorage hook is supposed to contain this logic

**Review by julialbq — APPROVED** [review]

File: src/atoms/Pun/Pun.test.js [hunk 6]
```diff
@@ -0,0 +1,68 @@
...
+      expect(contextCell).toBeInTheDocument();
+      expect(row).toContainElement(contextCell);
+
+      const messageCell = screen.queryByText(pun.message);
+      expect(messageCell).toBeInTheDocument();
+      expect(row).toContainElement(messageCell);
+
+      const likeButton = screen.queryAllByRole("button", { name: /like/i, container: row });
+      expect(likeButton.length).toBeGreaterThan(0);
+      expect(likeButton[0]).toBeInTheDocument();
```
Comment (gabrielcicero45) [inline src/atoms/Pun/Pun.test.js:65]: Consider making the components test separately. It's useful to identify where's the errors coming.

**Review by gabrielcicero45 — APPROVED** [review]

**Pull request merged**